lib/mix/lib/mix/tasks/deps.tree.ex

```diff
@@ -51,9 +51,7 @@ defmodule Mix.Tasks.Deps.Tree do
 
     if opts[:format] == "dot" do
       callback = callback(&format_dot/1, deps, opts)
-      Mix.Utils.write_dot_graph!("deps_tree.dot", "dependency tree", root, callback, opts)
-      Mix.shell.info "Generated \"deps_tree.dot\" in current directory.\n" <>
-                     "You can use http://www.graphviz.org/ to open it."
+      Mix.Utils.write_dot_graph!("deps_tree", "dependency tree", root, callback, opts)
```

Comment:
I don't think we should move the message to inside write_dot_graph!. If so, we should rename the function. For now, our only two use cases print similar messages but I am not sure if that will be true for future ones. I was already quite on the fence for renaming build into write but I don't think there would be any reason to keep the dot in memory. :)